ci/infra/azure/main.tf

```diff
@@ -0,0 +1,17 @@
+provider "azurerm" {
+  # version = "=2.5.0"
```

Comment:
Should we need this comment (I assume this is more for debugging purpose)? I believe the version of azurerm shuold be only offerred by our supported version.

Reply:
removed